### PR TITLE
Add window.open binding.

### LINKED ISSUE #19: Missing `window.open`
I'm not able to find the `window.open` method:
https://developer.mozilla.org/fr/docs/Web/API/Window/open

If it is implemented, would you point me to the right place in the docs? If not, would you like me to submit a pull request for this binding?

### PR DESCRIPTION
**Pull request by dboris**
Fixes #19.

### CODE REVIEW

File: src/brr.ml [hunk 1]
```diff
@@ -1645,6 +1645,9 @@ module Window = struct
 
   (* Operations *)
 
+  let open' ?(name = Jstr.empty) ?(features = Jstr.empty) w u =
+    Jv.to_option Fun.id @@ Jv.call w "open" [|Uri.to_jv u; Jv.of_jstr name; Jv.of_jstr features|]
```
Comment (dbuenzli) [inline src/brr.ml:1649]: That doesn't seem to fit 80 columns. It should.

File: src/brr.mli [hunk 2]
```diff
@@ -3138,6 +3138,12 @@ module Window : sig
 
   (** {1:ops Operations} *)
 
+  val open' : ?name:Jstr.t -> ?features:Jstr.t -> t -> Uri.t -> t option
```
Comment (dbuenzli) [inline src/brr.mli:3141]: I don't think using `Uri.t` here is the right choice. It should be either `Jstr.t` or `Uri.t option`  please double check with the docs of [`window.open`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open) that the type allows to specify everything that can.

**Review by dbuenzli — CHANGES_REQUESTED** [review]
I left a few comments but I'm honestly still undecided to actually provide this as I think it only leads to bad UX.

File: src/brr.mli (hunk 2 above)
Comment (dbuenzli) [reply src/brr.mli:3141]: Could you please double check the status of the returned object whether it's really compatible with `Window.t`. The docs mention a [`WindowProxy`](https://html.spec.whatwg.org/multipage/window-object.html#the-windowproxy-exotic-object) object.

Comment (dboris) [reply src/brr.mli]: Changed the `Uri.t` to `Jstr.t`.
I'm able to call `Window` functions with the `WindowProxy` object, so it behaves like a normal window object.

File: src/brr.ml (hunk 1 above)
Comment (dboris) [reply src/brr.ml]: Updated.

**Comment by dbuenzli** [comment]
Thanks ! Your patch is in as a870de16aea0b2d485c5a8507.

**Pull request closed without merging**